CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Added
+
+- pyln-testing: Fix FD leaks in BitcoinD. ([#7130])
```

Comment:
Changelog needs to be added in the commit not in the file as explained https://docs.corelightning.org/docs/coding-style-guidelines#changelog-entries-in-commit-messages.

Reply:
Thanks! Just learned about this :) Fixed here and will address in my other PRs as well.